themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fr.properties

```diff
@@ -0,0 +1,43 @@
+loadingMessage=Gestion du compte en cours de chargement ...
+fullName={0} {1}
+accountManagementWelcomeMessage=Bienvenue dans la Gestion de Compte Keycloak
+
+# Personal informations page
+personalInfoHtmlTitle=Informations Personnelles
+personalInfoIntroMessage=G\u00e9rez vos informations de base
+personalSubMessage=G\u00e9rez ces informations de base: votre pr\u00e9nom, nom de famille et email
+selectLocale=Choisissez une langue
+
+accountSecurityTitle=S\u00e9curit\u00e9 du Compte
+accountSecurityIntroMessage=G\u00e9rez votre mot de passe et l''acc\u00e8s \u00e0 votre compte
+applicationsIntroMessage=Auditez et g\u00e9rez les permissions d''acc\u00e8s des applications aux donn\u00e9es de votre compte
+
+# Authentication page
+signingIn=Authentification
+signingInSubMessage=Configurez les m\u00e9thodes d''authentification.
+basic-authentication=Authentification de Base
+password-display-name=Mot de passe
+password-help-text=Authentifiez-vous en saisissant votre mot de passe
+credentialCreatedAt=Cr\u00e9\u00e9 le
+two-factor=Authentification \u00e0 Deux Facteurs
+otp-display-name=Application d''authentification
+authenticatorActionSetup=Configurer
+otp-help-text=Entrez un code de v\u00e9rification \u00e0 usage unique fourni par l''application d''authentification.
+notSetUp={0} non configur\u00e9(e).
+
+# Device Activity page
+device-activity=Activit\u00e9 des Appareils
```

Comment:
Interesting you fixed this here, I was about to open a PR to fix the Account Security and Device Activity labels. in the `content.json`

However, the base theme could be inherited here, if the labels were to be set to  `deviceActivityHtmlTitle` and `accountSecurityHtmlTitle`.

Reply:
You are right.
I moved labels defined in theme `base` to the correct file.

Reply:
I think it would have been sufficient to update the `content.json` to use the already defined options in base,

Maybe a reviewer can voice a preference here.